### PR TITLE
Use main item image for PaymentConfirmation page.

### PR DESCRIPTION
**Pull request by nanu-c**
# Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Description

This fixes the main item image upload bug and finishes the implementation of showing the main item image on the payment confirmation page.

# Checklist:

- [ ] I have commented my code (or ChatGPT did), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings, neither in my IDE nor in my browser
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/views/PaymentConfirmation.vue [hunk 1]
```diff
@@ -3,30 +3,43 @@
...
-          <img class="rounded-full h-44 w-44 object-cover customborder border-4" alt="Titelblatt"
-            src="../assets/Titelseite.jpg">
+          <img
+            class="rounded-full h-44 w-44 object-cover customborder border-4"
+            alt="Titelblatt"
+            :src="
+              settStore.settings.MainItemImage
+                ? apiUrl + settStore.settings.MainItemImage
+                : '/Titelseite.jpg'
+            "
```
Comment (nanu-c) [inline src/views/PaymentConfirmation.vue:24]: The code is "show Titleseite.jpg if the mainItemImage isn't set", have you set a MainItemImage?

Comment (nanu-c) [reply src/views/PaymentConfirmation.vue:24]: @lebe1 ?

File: src/views/PaymentConfirmation.vue [hunk 2]
```diff
@@ -3,30 +3,43 @@
...
+          >
         </div>
       </div>
     </template>
   </component>
 </template>
 
 <script setup lang="ts">
-import { ref } from 'vue'
+import { ref, watch } from 'vue'
```
[the remark this answers is not in the document]
Comment (nanu-c) [reply src/views/PaymentConfirmation.vue:42]: That's true, i forgot that.

File: src/views/PaymentConfirmation.vue (hunk 1 above)
Comment (lebe1) [reply src/views/PaymentConfirmation.vue:24]: Nope I did not. Maybe we should implement a default MainItemImage in the backend for developing purposes then?

Comment (lebe1) [reply src/views/PaymentConfirmation.vue:24]: Works fine! So resolving this one

**Pull request merged**